protoc-gen-go/grpc/grpc.go

```diff
@@ -160,8 +160,9 @@ func (g *grpc) generateService(file *generator.FileDescriptor, service *pb.Servi
 	deprecated := service.GetOptions().GetDeprecated()
 
 	g.P()
-	g.P("// Client API for ", servName, " service")
-	g.P()
+	g.P(fmt.Sprintf(`// Client API for %s service.
```

Comment:
According to https://golang.org/doc/effective_go.html#commentary, this should probably be written as:

```go
// TestClient is the client API for the Test service.
//
// For semantics around resource cleanup, please refer to
// https://godoc.org/google.golang.org/grpc#ClientConn.NewStream.
```

Reply:
Also, shouldn't a similar comment be on the server equivalent of this?

Reply:
Done

And, although I think you're right, the server's case + recommendations are different and should probably live in a different PR.

Reply:
That's fine with me.